lehmann2008_evaporation/README.md

```diff
@@ -0,0 +1,29 @@
+# Lehmann et al. 2008 Fig. 8 Evaporation Dataset
+## Overview
+This artifact contains one dataset: a .csv containing data corresponding
+to the experiment displayed in figure 8 of Lehmann et al. 2008.
+
+This experiment involves measuring the evaporation rate of water from
+columns of varying heights, all containing coarse sand.
+The dataset contains timeseries of the evaporation rate [mm/day]
+throughout the duration of the experiment for each column height tested
+(150mm, 250mm, and 350mm).
+The experiment was run for 13 days, and the data is hourly starting 30 minutes
+into the simulation.
+For column heights of 150mm and 250mm, the last datapoint provided is at time
+12 days 23 hours. For column height 350mm, the last datapoint is at time
+12 days 18 hours.
+
+This data is used by ClimaLand.jl as a quantitative comparison for
+model-calculated evaporation calculated.
```

Comment:
This data is used by ClimaLand.jl as a quantitative comparison for
model-calculated evaporation.